src/main/java/org/betonquest/betonquest/compatibility/luckperms/permission/LuckPermsPermissionEvent.java

```diff
@@ -60,7 +60,7 @@ public void execute(final Profile profile) throws QuestRuntimeException {
         for (final Node node : buildNodes) {
             nodeApply.apply(data, node);
         }
-        userManager.saveUser(user);
+        luckPermsAPI.getMessagingService().ifPresent(service -> service.pushUserUpdate(user));
```

Comment:
For me it feels like now the normal update is missing, when there is no network?

And if it is the case, should there not be one method in LuckPerms doing booth?

Reply:
You're completely right, the saveUser should still be there. 

And Sadly no, there is not one Method that combines both, i already asked on their discord